### PR TITLE
Feature bl 9633 moth regression automation

### PR DESCRIPTION
**Pull request by karlbeckettbjss**

### CODE REVIEW

File: src/main/resources/features/moth/06 - dvla vehicle eu classification.feature
Comment (ibrahimjassat) [inline src/main/resources/features/moth/06 - dvla vehicle eu classification.feature]: Feature name doesn't make sense

File: src/main/resources/features/moth/08 - invalid registration.feature
Comment (ibrahimjassat) [inline src/main/resources/features/moth/08 - invalid registration.feature]: Don't forget to remove this

Comment (karlbeckettbjss) [reply src/main/resources/features/moth/08 - invalid registration.feature]: Removed

File: docs/step-definitions.md
Comment (ibrahimjassat) [inline docs/step-definitions.md:307]: Would like to have a second opinion of having this test/step in the AAT as it affects security of the project

File: drivers/readme.txt
Comment (ibrahimjassat) [inline drivers/readme.txt:1]: Should it not be the latest stable version

File: src/main/java/uk/gov/dvsa/mot/framework/WebDriverWrapper.java
Comment (ibrahimjassat) [inline src/main/java/uk/gov/dvsa/mot/framework/WebDriverWrapper.java]: typo 2nd

**Review by ibrahimjassat — CHANGES_REQUESTED** [review]

File: drivers/readme.txt
Comment (karlbeckettbjss) [reply drivers/readme.txt:1]: Nope, we are no longer keeping the chromedrivers in the repo, it will need to be installed seperatly via the instructions in the readme file

File: docs/step-definitions.md
Comment (karlbeckettbjss) [reply docs/step-definitions.md:307]: It's worth noting that all the MOTH tests already show how to get around the security of the page, but please do get a second opinion.

File: src/main/java/uk/gov/dvsa/mot/framework/WebDriverWrapper.java
Comment (karlbeckettbjss) [reply src/main/java/uk/gov/dvsa/mot/framework/WebDriverWrapper.java]: Fixed

File: src/main/resources/features/moth/18b - Print MOT certificates - PP.feature
Comment (ibrahimjassat) [inline src/main/resources/features/moth/18b - Print MOT certificates - PP.feature]: Remove before merging

File: src/main/resources/features/moth/18b - Print MOT certificates - PP.feature
Comment (ibrahimjassat) [inline src/main/resources/features/moth/18b - Print MOT certificates - PP.feature]: Remove before merging

File: src/main/resources/features/moth/18b - Print MOT certificates - PP.feature
Comment (ibrahimjassat) [inline src/main/resources/features/moth/18b - Print MOT certificates - PP.feature]: Same again

File: src/main/resources/features/moth/18b - Print MOT certificates - PP.feature
Comment (ibrahimjassat) [inline src/main/resources/features/moth/18b - Print MOT certificates - PP.feature]: Same again

File: src/main/resources/features/moth/18b - Print MOT certificates - PP.feature
Comment (ibrahimjassat) [inline src/main/resources/features/moth/18b - Print MOT certificates - PP.feature]: Same again

File: src/main/resources/features/moth/18b - Print MOT certificates - PP.feature
Comment (ibrahimjassat) [inline src/main/resources/features/moth/18b - Print MOT certificates - PP.feature]: Same again

**Review by ibrahimjassat — CHANGES_REQUESTED** [review]

File: docs/step-definitions.md
Comment (karlbeckettbjss) [reply docs/step-definitions.md:307]: Any update to this as I would like to get these changes merged in

**Pull request merged**
